examples/adeo-kafka-request-reply-asyncapi.yml

```diff
@@ -239,14 +198,10 @@ components:
             type: string
             format: date-time
             description: Technical timestamp for the costing calculation
-      bindings:
-        kafka:
-          key:
-            $ref: "https://deploy-preview-921--asyncapi-website.netlify.app/resources/casestudies/adeo/CostingResponseKey.avsc"
```

Comment:
this part need to be kept

Reply:
Feel free to open a new PR. 
I had to merge asap because CLI was affected, we don't know if other tools as well. The fix for that is also a wip so it doesnt happen again.